src/appleseed/renderer/modeling/frame/frame.cpp

```diff
@@ -899,15 +909,49 @@ DictionaryArray FrameFactory::get_input_metadata()
             .insert("default", "off")
             .insert("on_change", "rebuild_form"));
 
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "prefilter_spikes")
+            .insert("label", "Prefilter Spikes")
+            .insert("type", "enumeration")
+            .insert("items",
+                Dictionary()
+                    .insert("Off", "off")
+                    .insert("On", "on"))
+            .insert("use", "required")
+            .insert("default", "on")
+            .insert("visible_if",
+                Dictionary()
+                    .insert("denoiser", "on")));
+
+    metadata.push_back(
+        Dictionary()
+            .insert("name", "spike_threshold")
+            .insert("label", "Spike Thereshold")
+            .insert("type", "text")
+            .insert("use", "optional")
+            .insert("min",
+                Dictionary()
+                    .insert("value", "0.1")
+                    .insert("type", "hard"))
+            .insert("max",
+                Dictionary()
+                    .insert("value", "4.0")
+                    .insert("type", "hard"))
```

Comment:
`"hard"` means that there's absolutely no way to go above that value. Is that what we want?